### PR TITLE
[enocean] Add support for Eltako TF-AHDSB

### LINKED ISSUE #4582: [enocean] Add support for Eltako TF-AHDSB
The outdoor brightness sensor Eltako TF-AHDSB has two differnt modes:

FAH mode
Sends brightness data.
EEP A5-06-01
DB3 = 0x00..0x64 = 0-50 lux, linear (brightness, only if DB2 = 0x00)
DB2 = 0x00..0xFF = 300..30000 lux, linear (brightness)
DB1 = -
DB0 = 0x08

TF-Mode (additional to FAH mode)
Sends on / off telegrams when a given brightness is reached.
EEP A5-38-08
DB3 = 0x01 (-> CMD 1 "switching")
DB0 = (0x08 = off) || (0x09 = on) || (0x28 = unlock)

See https://github.com/fruggy83/openocean/issues/33

### PR DESCRIPTION
**Pull request by dominikkv**
Both EEPs implemented.

Signed-off-by: Dominik Krickl-Vorreiter dominikkv@gmx.de
closes #4582
See https://github.com/fruggy83/openocean/pull/44

### CODE REVIEW

File: addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2010-2019 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (wborn) [inline addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01.java]: Please use the new EPL-v2.0 header. See also this [community topic](https://community.openhab.org/t/tutorial-update-license-to-epl-v2-0-of-your-binding-in-development-or-pull-request/65088?u=wborn).

File: addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01_ELTAKO.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2019 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
```
Comment (wborn) [inline addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01_ELTAKO.java]: Please use the new EPL-v2.0 header. See also this [community topic](https://community.openhab.org/t/tutorial-update-license-to-epl-v2-0-of-your-binding-in-development-or-pull-request/65088?u=wborn).

File: addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01.java (hunk 1 above)
Comment (dominikkv) [reply addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01.java]: Done, thanks!

File: addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01_ELTAKO.java (hunk 2 above)
Comment (dominikkv) [reply addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_06/A5_06_01_ELTAKO.java]: Done, thanks!

**Comment by dominikkv** [comment]
Thanks @wborn, I updated the header. If I can do anything more, please let me know.

**Pull request merged**